src/ContosoTraders.Ui.Website/tests/api.cart.spec.ts

```diff
@@ -10,7 +10,7 @@ const PRODUCTQUANTITY = 1;
 
 // SETUP: Create a new cart
 test.beforeAll(async ({ request }) => {
-    const newCart = await request.post(`${process.env.REACT_APP_APIURLSHOPPINGCART}/ShoppingCart`, {
```

Comment:
this is separate from the baseURL (it's for the ACA shopping cart API), so using a relative path here won't work.

Reply:
see test failures here: https://github.com/microsoft/contosotraders-cloudtesting/actions/runs/4746674582/jobs/8430895228

Reply:
changed it to test.use which should work and seems prettier here.

Reply:
great idea!